miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppPermission.kt

```diff
@@ -1,9 +1,9 @@
-package com.rakuten.tech.mobile.miniapp.js
+package com.rakuten.tech.mobile.miniapp.permission
 
 /** Type of miniapp permission. **/
-enum class MiniAppPermissionType(val type: String) {
-    UNKNOWN("unknown"),
-    LOCATION("location");
+enum class MiniAppPermissionType(val type: String, val platform: String) {
```

Comment:
Instead of adding a `platform` type, maybe we should instead create a separate `MiniAppCustomPermissionType` type for the custom permissions and just use `MiniAppPermissionType` for device permissions? It seems like that would be simpler and easier to adapt to how we want to use it. 

For example, the device permissions are treated differently than custom permissions by the host app (as they have to implement a custom dialog for custom permissions) and the Mini App is able to request multiple custom permissions at once.